services/worker/src/worker/main.py

```diff
@@ -123,6 +136,35 @@ def kill_zombies(self) -> None:
             Job.objects(pk__in=[zombie.pk for zombie in zombies]).update(
                 status=Status.ERROR, finished_at=get_datetime()
             )
+            for zombie in zombies:
+                job_info = JobInfo(
+                    job_id=str(zombie.pk),
+                    type=zombie.type,
+                    dataset=zombie.dataset,
+                    config=zombie.config,
+                    split=zombie.split,
+                    force=zombie.force,
+                    priority=zombie.priority,
+                )
+                job_runner = self.job_runner_factory.create_job_runner(job_info)
```

Comment:
it's a bit weird to create a new job runner, just to update the cache?

Reply:
Yea but I need it to be able to call `.get_version()` and `.get_dataset_git_revision()`